.github/workflows/prove.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: Perl tests
+# yamllint disable-line rule:truthy
+on: [push, pull_request]
+jobs:
+  fullstack:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/ci/containers/base:latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: bash -x tools/ci/build_cache.sh
+      - name: Fix permissions of working folder
+        run: sudo chown -R squamata ..
```

Comment:
Is this even necessary for GHA? If it's necessary here as well I suggest to not duplicate but include as part of one of the shell scripts

Reply:
Yes, otherwise the `git clone` in `build_autoinst.sh` fails

Reply:
Not sure it feels very clean to handle permissions in `build_autoinst.sh`... but I guess it might work